### PR TITLE
Add page for PyDay 2019

### PR DESCRIPTION
**Pull request by ifosch**
This is the base for the page.
More stuff is to be added as the event gets organized.

### CODE REVIEW

**Comment by DZPM** [comment]
Needs the logos for all sponsors

**Comment by ifosch** [comment]
> Needs the logos for all sponsors

I just added them.

**Comment by ifosch** [comment]
He afegit el logo de Satellogic que hi ha al drive, i d'altres patrocinadors, però treient-los d'altres edicions i events.

File: content/pyday-bcn-2019/contents.lr [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+PyDay Barcelona 2019 has the aim of expanding the local Python community.
+
+# Stay tuned
+If you want to keep up to date with all the news related to this event, follow us on Twitter at [@pybcn](https://twitter.com/pybcn).
+
+## Calling for Proposals
+Are you interested in giving a workshop?
+We want YOU!
+
+Please, fill this [form](https://meet.meetup.com/wf/click?upn=pEEcc35imY7Cq0tG1vyTt6eARwsW6FpLo-2FVDoCXxpa-2FZ8voU5V4txYS1iYFrEaQmWxLz71WjU6TQ0R7eMiPG8AWCp97xGm1CCFQGdi9QAE45hS9LKDwys5rHRidIkYbgkUWzEJeMgenRCBaBZ1LkmlvneCTFWvRayfePWz-2F7Vlbn-2B9GEbYlRc5MUYsGi4DjPJomXcUGRtz3wDcQASj2405Qo27OJx6NIEKquCXk-2BIfnLFqt0FjgZ52ymVUIhyqPhKJfF-2FVm4By1elwMqic1gEtSHeUBXPIxCqNM0md7sZX-2Fqf44JvZoP31T9QtkJL62-2FykqgLtCqm5FsX-2Bcs-2Bq2wEeEGgv2Gp2x8DexotPDiWcZ5f5Ri2hOBZ1DrTMdMipBBuq9BnMoUuwbAgLPscKlzNNR1684wClnVAfJ4-2B9RkPbW0C2MRv9qIyS9e-2Fxp1mLHYIcE5s6IUYEzKOa6BBhU5eQ-3D-3D_w0RjhZcfChRE64x7dG2Ln0MCqK81FTZitGqDUz-2Fr9vcODiUpNnFwVsrSNCbDvQ-2FQbGjGuDAPI7FaXRvz1WdqiglV9uDJIGeqH4W7AJ7PI4dDfY0NZemTam0q285G5TFHfMg6g2aZGeqPK27QISQ5kkCYO0OJrdLQGmcD-2FdQnTd5ykKF0U5tQLZj-2FbWGrtppYbbuetcv4CmNIhiLrX7j3Ruo5erm0ck0iYmn1fSOL4UA-3D) with your proposal details.
```
[the remark this answers is not in the document]
Comment (ifosch) [reply content/pyday-bcn-2019/contents.lr]: Fixed

**Comment by Juanlu001** [comment]
Missing link in the header? :innocent:

**Comment by ifosch** [comment]
> Missing link in the header?

@Juanlu001 This is coming now, in a separate PR.

**Comment by ifosch** [comment]
Checkout #33 
cc/ @lpmayos @DZPM @NataliaSirera @Juanlu001

**Pull request merged**